README.md

```diff
@@ -1,6 +1,23 @@
-# Affinidi Trust Development Kit (Affinidi TDK)
+<a id="top"></a>
+# Affinidi Trust Development Kit (TDK) for .NET
+
+The Affinidi Trust Development Kit (TDK) empowers developers to streamline the integration of Affinidi services into their applications with comprehensive libraries and tools, boosting their productivity. 
+This toolkit also makes it easy to easily and integrate [Affinidi Elements](https://docs.affinidi.com/docs/affinidi-elements/) and [Frameworks](https://docs.affinidi.com/frameworks/iota-framework/) into your application. It minimises dependencies and enables developers to integrate seamlessly into the [Affinidi Trust Network (ATN)](https://docs.affinidi.com/docs/).
```

Comment:
> makes it easy to easily and integrate

Is this correct?

> It minimises dependencies and enables developers to integrate seamlessly into the [Affinidi Trust Network (ATN)](https://docs.affinidi.com/docs/).

(suggestion to go with) It enables developers to integrate seamlessly into the [Affinidi Trust Network (ATN)](https://docs.affinidi.com/docs/).